packages/static/src/tabs/tab/index.tsx

```diff
@@ -0,0 +1,183 @@
+    const tabRef = useDOMRef<HTMLDivElement>(ref)
+    const isSelected = React.useMemo(
+      () => currentKey === item.key,
+      [currentKey],
+    )
+    const isDisabledItem = React.useMemo(
+      () => (item.key ? [...disabledKeys].includes(item.key) : false),
+      [disabledKeys],
+    )
+    const disabledState = isDisabled || isDisabledItem
+
+    const handleSelect = () => {
+      if (item.key && !disabledState) onSelect(item.key)
+    }
+
+    const handleKeyDown = (event: React.KeyboardEvent<HTMLDivElement>) => {
+      const key = event.key
+      if (key === 'Enter' || key === ' ') {
+        event.preventDefault()
+        handleSelect()
+      }
+    }
+
+    const { useDescendant } = useKeyboardNavigationState()
+    const { index, register } = useDescendant({ disabled: disabledState })
+    const { onFocus } = useKeyboardNavigation()
+
+    const handleOnFocus = () => {
+      onFocus?.(index)()
+    }
+
+    const mergeRef = useMergeRefs([tabRef, register])
+    const tabClassName = [
+      `cdg-tab-item-wrapper`,
+      styles.tab,
+      styles[`icon${icon.charAt(0).toUpperCase() + icon.slice(1)}`],
+      styles[`${variant}`],
+      styles[`active${isSelected ? 'True' : 'False'}`],
+      styles[`disabled${!!disabledState ? 'True' : 'False'}`],
+      styles[variant + `Disabled${!!disabledState ? 'True' : 'False'}`],
+      styles[variant + `Active${!!isSelected ? 'True' : 'False'}`]
+    ].filter(Boolean)
+      .join(' ')
+
+    const tabIconClassName = [
+      styles.icon,
+      styles[`icon${icon.charAt(0).toUpperCase() + icon.slice(1)}Icon`],
+      variant && styles[variant + `Icon`],
+      variant && disabledState && styles[variant + `Disabled${!!disabledState ? 'True' : 'False'}Icon`],
+      variant && isSelected && styles[variant + `Active${!!isSelected ? 'True' : 'False'}Icon`]
+    ].filter(Boolean)
+      .join(' ')
+
+    const customCss = {
+      [styles.simple]: {
+        '&:focus-visible': {
+          boxShadow: `0px -2px ${indicatorColor}`,
+          color: `${textColor}`,
+        },
+        '&:hover': {
+          color: `${textColor}`,
+        },
+      },
+      [styles.h5]: {
+        '&:focus-visible': {
+          boxShadow: `0px -2px ${indicatorColor}`,
+        },
+      },
+      [styles.simpleActiveTrue]: {
+        borderBottom: `${indicatorColor} 2px solid`,
+        color: `${textColor}`,
+      },
+      [styles.simpleActiveTrueIcon]: {
+        backgroundColor: `${textColor}`,
+        '& path': {
+          fill: `${textColor}`,
+        },
+      },
+      [styles.h5ActiveTrueIcon]: {
+        backgroundColor: `${textColor}`,
+      },
+      [styles.rounded]: {
+        color: `${textColor}`,
+      },
+      [styles.icon]: {
+        backgroundColor: `${textColor}`,
+      },
+      [styles.activeTrue]: {
+        backgroundColor: `${textColor}`,
+      },
+      ...css as Object,
+    }
+
+    return (
+      <CssInjection css={customCss} childrenRef={mergeRef}>
```

Comment:
you do this then the users won't be able to pass their own css through this prop anymore

Reply:
css object for user pass in the end of customCss